src/main/java/uk/gov/justice/digital/delius/transformers/OffenderTransformer.java

```diff
@@ -316,4 +305,147 @@ private Human humanOf(String forename, String forename2, String surname) {
     }
 
 
+    public List<uk.gov.justice.digital.delius.data.api.ResponsibleOfficer> responsibleOfficersOf(Offender offender, boolean current) {
+
+        ArrayList<ResponsibleOfficer> responsibleOfficers = new ArrayList<>();
+
+        if (offender.getOffenderManagers() != null && !offender.getOffenderManagers().isEmpty()) {
+
+            for (uk.gov.justice.digital.delius.jpa.standard.entity.OffenderManager om : offender.getOffenderManagers()) {
```

Comment:
Can we do this in a Streaming style please?

Reply:
We certainly can. Working on it.